.azure-pipelines/client.yml

```diff
@@ -87,6 +87,18 @@ jobs:
       vmImage: '$(OSVmImage)'
 
     steps:
+      - powershell: |
+          Invoke-WebRequest -Uri "https://github.com/chidozieononiwu/azure-sdk-for-net/releases/download/verifyAgentOS_v1.1/verify_agent_os.zip" `
+          -OutFile "verify_agent_os.zip" | Wait-Process; Expand-Archive -Path "verify_agent_os.zip" -DestinationPath "./azure-sdk-tools/"
+        workingDirectory: '$(System.DefaultWorkingDirectory)'
```

Comment:
We should be careful about putting our tools in here as this is the same place that the source code is cloned into and we don't want the contents of our tools to interfere with that. I suggest using `Build.BinariesDirectory`, see https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#build-variables for more information.

Reply:
Ok, I will update that.